packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -496,7 +496,7 @@ export class MetriportMedicalApi {
     resources?: readonly ResourceTypeForConsolidation[],
     dateFrom?: string,
     dateTo?: string,
-    conversionType?: string,
+    conversionType = "json",
```

Comment:
Reason im setting a default value here is because if i make this required without one then it needs to go above the optional params due to our linter. To avoid the breaking change on the sdk for all cxs since this isnt an object will just make the conversionType default to json so the API will always receive a conversionType even if they dont explictly set one.

Reply:
Sounds good. Let's just update the TSDoc, please? It's saying defaults to JSON in the body of the WH request.